Resources/Prototypes/Entities/Structures/Doors/Airlocks/access.yml

```diff
@@ -650,5 +650,23 @@
   - type: AccessReader
     access: [["External"]]
 
+#HighSecDoors
+- type: entity
+  parent: HighSecDoor
+  id: HighSecCommandLocked
+  suffix: Command, Locked
+  components:
+  - type: AccessReader
+    access: [["Command"]]
+  - type: Wires
+    LayoutId: HighSec
 
-
+- type: entity
+  parent: HighSecDoor
+  id: HighSecCaptainLocked
+  suffix: Captain, Locked
+  components:
+  - type: AccessReader
+    access: [["Captain"]]
+  - type: Wires
```

Comment:
Does it need to have Wires here? The HighSecDoor parent has it

Reply:
yea youre right i was just following the conventions the other doors were using